swift/torchacc_utils.py

```diff
@@ -30,6 +30,15 @@ def get_bucket_sizes(max_length: int) -> List[int]:
     if os.getenv('TORCHACC_DATA_BUCKETS') is not None:
         bucket_sizes = [int(x) for x in os.getenv('TORCHACC_DATA_BUCKETS').split(',')]
         bucket_sizes.append(max_length)
+    elif os.getenv('TORCHACC_CACHE_PATH') is not None: # padding strategy when persistent cache is enabled
```

Comment:
Could we replace the `else` block with this logic as defualt bucketing strategy?

Reply:
Fixed.